specification/schemas/Invoice.json

```diff
@@ -0,0 +1,93 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalInvoice"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "description": {
+              "oneOf": [
+                {
+                  "type": "string",
```

Comment:
You can use `"type":  ["string", "null"]`, instead of the oneOf when the type is a simple string or number.